import-chrome/src/import-chrome.cpp

```diff
@@ -133,11 +133,30 @@ int main( int argc, char** argv )
     printf( "\33[2KParsing...\r" );
     fflush( stdout );
 
+    struct PidTid {
+      uint64_t tid;
+      uint64_t pid;
+      uint64_t both; // fake thread id, unique within Tracy
+    };
+
+    std::vector<PidTid> tids;
     std::vector<tracy::Worker::ImportEventTimeline> timeline;
     std::vector<tracy::Worker::ImportEventMessages> messages;
     std::vector<tracy::Worker::ImportEventPlots> plots;
     std::unordered_map<uint64_t, std::string> threadNames;
 
+    const auto getTid = [&](uint64_t pid, uint64_t tid) -> uint64_t {
+        for ( auto &pair : tids ) {
+            if ( pair.pid == pid && pair.tid == tid ) {
+                return pair.both;
+            }
+        }
+
+        const auto result = tids.size();
+        tids.emplace_back(PidTid {.tid=tid, .pid=pid, .both=result});
```

Comment:
This is a C++20 feature.

Reply:
oh, my bad. does `push` work?

Reply:
You should do `tid.emplace_back( PidTid { tid, pid, result } );`, in the struct order. Designated initializers are not in the standard.